csharp/samples/getting-started-receiving.md

```diff
@@ -0,0 +1,201 @@
+# Get started with Event Hubs in .NET
```

Comment:
># [](start = 0, length = 1)

Can we use numbering such as below instead of dashes?
1.
1.1.
1.2.
2.
2.1.
2.2.